### DISCUSSION TITLE
주말 동안 테마 팔레트를 픽해주세요!

### DISCUSSION THREAD

**Discussion by gomiseki** [FE-CheckList-Gallery/FE-CheckList-Gallery#2] [Announcements]
# 테마 색 공모전

[어도비의 팔레트 추천 사이트](https://color.adobe.com/ko/explore?page=2) 처럼 테마색 모음을 제공하는 레퍼런스는 많습니다.
**주말 동안** **전시회**에 맞는 테마 색을 **최소한 1개** 이상 **hexcode 형식**으로 올려보고 투표까지 진행해볼게요!

**_댓글로 남겨주세요_**

참고로 백틱(`)안에 hexcode를 입력하면  <br>

`#2D2D2D`

다음과 같이 약간의 미리보기를 제공합니다.

**Comment by ycs1m1yk** [comment]
저는 우리 갤러리에 품격을 더해줄 `#318CE7` `(Bleu de France)`를 추천합니다! 
> Bleu de France,  프랑스를 대표하는 색

갤러리 -> 예술 -> 예술의나라 프랑스 -> 프랑스하면 파란색

![image](https://user-images.githubusercontent.com/27720475/173114029-a0697176-99ac-40c9-bd65-8ca7aead8f69.png)

**Reply by ycs1m1yk** [reply]
손흥민 선수의 epl 득점왕 기념으로 토트넘의 팀컬러인 `#132257` 도 추천해봅니다 :)

![image](https://user-images.githubusercontent.com/27720475/173114639-65184a81-262b-4802-a58d-35604197ba31.png)

출처: https://teamcolorcodes.com/tottenham-hotspur-colors/

**Comment by gomiseki** [comment]
<img src="https://user-images.githubusercontent.com/50083131/173169204-0b825dd6-93ac-44ec-aa0a-6dd842f78838.png" width="300">
갤럭시 너낌을 찾다가 사진의 왼쪽 5개가 예뻐서 들고와봤습니다.

-  `#2D2D2D`
- `#6C4F85`
- `#BFC1CE`
- `#7D95C1`
- `#EFF0EB`

**Reply by ycs1m1yk** [reply]
@gomiseki  투표는 한번에하는건 어떨까요? 화요일 8시에 모였을때 투표하는 시간 가지면 될까요

**Comment by sori-sodong** [comment]
저도 파란계열 예쁜 것 같아요.
`#3949ab`
`#00227b`
`#6f74dd`
제안합니다!

**Comment by minsu2344** [comment]
저도 여름이라 바다색으로 정해봤어요!
`#00ccff`
`#00eeff`
`#A1FFFB`
`#66a5ad`
`#c4dfe6`

**Comment by SeolJaeHyeok** [comment]
다른 분들이 파란색 계열로 선택을 많이 해주셔서 저는 다른 색 계열로 한 번 찾아봤습니다!

색상 관련된 레퍼런스를 찾아서 참조해보니 창의성과 관련된 색은 보라색 계열인 걸 확인했습니다.
![color-guide](https://user-images.githubusercontent.com/71430291/173222716-b8c7a7ed-4a6f-4f40-bf8b-dbc68fb6be45.png)



Frontend가 사용자와 마주치는 공간이고 디자이너와의 협업을 통해 창의적인 결과를 도출하는 과정이라고 생각을 해서 
`Creative`를 뜻하는 색상들을 몇개 골라봤습니다:)

`#f1007d`
`#1f155a`
`#6b397b`
`#d3117f`
`#d42b89`
`#940299`
`#e00270`
